buildroot/package/con2fbmap/con2fbmap.mk

```diff
@@ -0,0 +1,25 @@
+#############################################################
+#
+# con2fbmap system
+#
+#############################################################
+
+
+CON2FBMAP_VERSION = 1.0
+CON2FBMAP_SITE = $(TOPDIR)/../con2fbmap
```

Comment:
Probably makes sense to move the conf2bmap files down into this subdirectory, rather than adding an extra directory at the top-level? (but it's been a long time since I've messed around with Buildroot, so accept my apologies if that wouldn't work!)

Reply:
I put it at the same level as my rebootp package in PINN, keeping it separate from buildroot.

Reply:
Fair enough, I'll defer this decision to @maxnet / @XECDesign

Reply:
>Fair enough, I'll defer this decision to @maxnet / @XECDesign

Not mine to make.
Just dropping by here as normal bystander, as someone mentioned my name :-)
Have not been involved with NOOBS in recent years.

Reply:
> Just dropping by here as normal bystander, as someone mentioned my name :-)

:slightly_smiling_face: :wave: 

> Have not been involved with NOOBS in recent years.

Ditto. I was only mentioning your name because you have more recent buildroot experience than me. Apologies if I gave the wrong impression.

Reply:
> 
> 
> Probably makes sense to move the con2fbmap files down into this subdirectory, rather than adding an extra directory at the top-level? (but it's been a long time since I've messed around with Buildroot, so please accept my apologies if that wouldn't work!)

Same for any weird reviews from me. I'll keep non tech comments separate to make it easy. 

But those non tech ones may end up to be tech comments if you think about it.